x/precisebank/README.md

```diff
@@ -354,3 +378,195 @@ $$b'(R) - b(R) = \begin{cases} 0 & r' > r \land f'(1) < f(1) \\
 1 & r' > r \land f'(1) \geq f(1) \\
 0 & r' \leq r \land f'(1) \geq f(1) \\
 \end{cases}$$
+
+## State
+
+The `x/precisebank` module keeps state of the following:
+1. Account fractional balances.
+2. Remainder amount. This amount represents the fractional amount that is backed
+   by the reserve account but not yet in circulation. This can be non-zero if
+   a fractional amount less than `1ukava` is minted.
```

Comment:
Should we add a note about current mainnet state and remainder is always zero (we think) at the end of each block due to the EVM always using matching mint/burns?